sdk/utils.ts

```diff
@@ -1,12 +1,38 @@
 import fetch, { RequestInit, Response } from 'node-fetch';
+import { ITraitConfig } from './types';
 // @ts-ignore
 if (typeof fetch.default !== 'undefined') fetch = fetch.default;
 
-export function generateIdentitiesData(identifier: string, traits: { [key: string]: any }) {
-    const traitsGenerated = Object.entries(traits).map(trait => ({
-        trait_key: trait[0],
-        trait_value: trait[1]
-    }));
+type Trait = {
+    [key: string]: ITraitConfig | any
+}
```

Comment:
This type represents the mapping containing the identity traits, so the name should be plural, I believe.

Besides that, while it makes an illustrative point, a union with `any` means `any`, so there won't be any difference until the type guard is used.

```suggestion
type Traits = {[key: string]: any}
```

Reply:
Corrected